gpiozero/boards.py

```diff
@@ -1185,3 +1185,86 @@ def on(self):
     def off(self):
         self.value = False
 
+class SevenSegmentDisplay(LEDBoard):
```

Comment:
if I want to know if it is on or off, how do?

I suggest a status attribute for it